py/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@llama_cloud_services/py",
+  "version": "0.6.55",
+  "private": "true",
+  "license": "MIT",
+  "scripts": {
+    "version": "cd .. && pnpm changeset version && bash py/scripts/new_version.sh",
```

Comment:
this shouldn't call `changeset version`, just the sync version script - `changeset version` must be called by the main monorepo script (see comment below).